tutorials/cbt-import-csv.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Import a CSV file into a Cloud Bigtable table
+description: Learn how to import a CSV file into a Cloud Bigtable table.
+author: thebilly
+tags: Cloud Bigtable, Dataflow, Java
+date_published: 2018-06-26
+---
+
+
+
+## Prerequisites
+
+### Install software and download sample code
+
+
+Make sure you have the following software installed:
+
+- [Git](https://help.github.com/articles/set-up-git/)
+- [Java SE 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Apache Maven 3.3.x or later](https://maven.apache.org/install.html)
+       
+    > If you haven't used Maven before check out this [5 minute quickstart](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html)
+
+### Set up your Cloud Project
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1.  Install the **[Google Cloud SDK](https://cloud.google.com/sdk/)** if you do
+    not already have it. Make sure you
+    [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK. Use
+    your project's ID to set the default project for the `gcloud` command-line tool. 
+
+## Upload your CSV
```

Comment:
Given this is a tutorial, it would be good to have a sample csv that can be used here to showcase this dataflow job. Else, it reads more like the README for the dataflow job. Tutorials should ideally ensure a path to success for the user.

Reply:
Sounds good